examples/vm_snapshot_removal_based_on_timestamp.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Example - delete all snapshots that are older than X days.
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  environment:
+    SC_HOST: "your_host"
+    SC_USERNAME: "your_username"
+    SC_PASSWORD: "your_password"
+  vars:
+    # Format: 'YYYY-MM-DD hh:mm:ss'
+    # All snapshots older than this date will be deleted.
+    use_date: '2023-05-03 12:52:00'
+
+  tasks:
+  # ------------------------------------------------------
+    - name: List all snapshots
+      scale_computing.hypercore.vm_snapshot_info:
+      register: snapshot_results
+
+    - name: Convert date to unix timestamp 'epoch'
+      set_fact:
+        epoch_timestamp: "{{ (use_date | to_datetime).strftime('%s') }}"
```

Comment:
Timezone could be problematic here.

I tried to add timezone to date. Test task:
```
    - name: Show date
      ansible.builtin.debug:
        msg: |
          {{ ('2023-05-03 12:52:00' | to_datetime).strftime('%s') }}
          {{ ('2023-05-03 12:52:00+0000' | to_datetime('%Y-%m-%d %H:%M:%S%z')).strftime('%s') }}
          {{ ('2023-05-03 12:52:00+0200' | to_datetime('%Y-%m-%d %H:%M:%S%z')).strftime('%s') }}
          {{ ('2023-05-03 12:52:00 UTC' | to_datetime('%Y-%m-%d %H:%M:%S %Z')).strftime('%s') }}
          {{ ('2023-05-03 12:52:00 CEST' | to_datetime('%Y-%m-%d %H:%M:%S %Z')).strftime('%s') }}
          {{ ('2023-05-03 12:52:00 CET' | to_datetime('%Y-%m-%d %H:%M:%S %Z')).strftime('%s') }}
#          {{ ('2023-05-03 12:52:00 EST' | to_datetime('%Y-%m-%d %H:%M:%S %Z')).strftime('%s') }}
#          {{ ('2023-05-03 12:52:00 EDT' | to_datetime('%Y-%m-%d %H:%M:%S %Z')).strftime('%s') }}
#          {{ ('2023-05-03 12:52:00 America/Los_Angeles' | to_datetime('%Y-%m-%d %H:%M:%S %Z')).strftime('%s') }}
```

Last 3 lines generated error for me (ansible 2.14.5, python 3.10). Output from other lines was not what I would expect:
```
TASK [Show date] ****************************************************
ok: [localhost] => 
  msg: |-
    1683111120
    1683111120
    1683111120
    1683111120
    1683111120
    1683111120
```

Looks like timezone info is accepted, and then just ignored :/.

Can we document that `use_date` should be specified in UTC timezone? Is it UTC?

If needed, we can later use some code from check_local_time role, and do our own timezone aware conversion.

Reply:
Well, the main thing is that the time specified in the ansible playbook has to match the timezone set on the Scale cluster.
As long as that is the case there shouldn't be any timezone limitations set on our side.
We can add a comment that says something like "use_date timezone should match Scale cluster timezone".